tests/routes/test_settings.py

```diff
@@ -36,7 +36,7 @@ def test_reset_restores_defaults(self):
 
         # Reset session
         response = self.app.post('/settings/reset').data
-        self.assertIn(b'The example app space v1', response)
+        self.assertIn(b'Loaded from local file', response)
```

Comment:
Loaded from local file?

Reply:
That's what the html says if you're loading the session locally

Reply:
Where do you change that test to evaluate that instead of the space name?

Reply:
ah I guess it evaluates the full string of that form?

Reply:
Yes, it reads the entire html response as a string and verifies some minor things.